packages/graphql-mocks/test/unit/graphql/handler.test.ts

```diff
@@ -276,5 +276,13 @@ Syntax Error: Unexpected Name "NOT"`);
     expect(context.fromQueryContext, 'query context is spread into context').to.exist;
     expect(context.pack, 'is defined in context').to.exist;
```

Comment:
Here there are already tests that ensure that `pack` is included on context in a query.